UI/DragNDrop.h

```diff
@@ -5,7 +5,7 @@
 class DragNDrop : public kitten::ClickableUI
 {
 protected:
-	bool m_isDragging=false, m_backToOrigin= false;
+	bool m_isDragging, m_backToOrigin= false;
```

Comment:
Can you add `m_isDragging(false)` in the constructor of `DragNDrop.cpp` ?

Reply:
done